### ISSUE TITLE
WARNING: importing deprecated binding Colors.RGB1 into ImageCore.

### ISSUE THREAD

**Issue by prbzrg** [JuliaIO/ImageMagick.jl#220]
I got these warnings in the precompilation:

```julia
┌ ImageMagick [6218d12a-5da1-5696-b52f-db25d2ecc6d1]
│  WARNING: importing deprecated binding Colors.RGB1 into ImageCore.
│  WARNING: importing deprecated binding Colors.RGB4 into ImageCore.
└
```

**Comment by ViralBShah** [comment]
Removing ImageCore 0.8 in the Project.toml seems to help fix this.

**Comment by ViralBShah** [comment]
Fixed in 1.3.1
